docs/articles_en/openvino_workflow/running_inference_with_openvino.rst

```diff
@@ -13,7 +13,7 @@ Running Inference with OpenVINO™
    openvino_docs_OV_UG_Working_with_devices
```

Comment:
build error indicates that openvino_docs_OV_UG_Working_with_devices and openvino_docs_ov_ug_supported_plugins_gna contain a link to deleted article openvino_docs_OV_UG_model_state_intro, please delete all references to this article